modules/3d/src/levmarq.cpp

```diff
@@ -340,20 +340,20 @@ LevMarq::Report detail::LevMarqBase::optimize()
 
     bool found = (smallGradient || smallStep || smallEnergyDelta || smallEnergy);
 
-    CV_LOG_INFO(NULL, "Finished: " << (found ? "" : "not ") << "found");
+    CV_LOG_DEBUG(NULL, "Finished: " << (found ? "" : "not ") << "found");
     std::string fr = "Finish reason: ";
     if (settings.checkMinGradient && smallGradient)
-        CV_LOG_INFO(NULL, fr + "gradient max val dropped below threshold");
+        CV_LOG_DEBUG(NULL, fr + "gradient max val dropped below threshold");
```

Comment:
In case if debug logs are disabled the macro is empty. You'll get something line if (settings.checkMinGradient && smallGradient); Please add {} for block to fix compiler warnings.